### PR TITLE
nixfmt: unstable-2024-05-28 -> unstable-2024-07-03

### PR DESCRIPTION
**Pull request by infinisil**
## Description of changes

Updates nixfmt to include https://github.com/NixOS/nixfmt/pull/210, needed to unblock https://github.com/NixOS/nixpkgs/pull/322537

Ping @NixOS/nix-formatting for review and merge

## Things done
- [x] Built on `x86_64-linux`

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/ni/nixfmt-rfc-style/generated-package.nix [hunk 1]
```diff
@@ -22,7 +22,7 @@ mkDerivation {
   ];
   jailbreak = true;
   homepage = "https://github.com/NixOS/nixfmt";
-  description = "Opinionated formatter for Nix";
+  description = "An opinionated formatter for Nix";
```
Comment (eclairevoyant) [inline pkgs/by-name/ni/nixfmt-rfc-style/generated-package.nix:25]: The reason this was removed was due to https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes:

> `meta.description` must ... [n]ot start with the definite or an indefinite article

Comment (infinisil) [reply pkgs/by-name/ni/nixfmt-rfc-style/generated-package.nix]: Ahh, that makes sense now, thanks! I updated it here and made a PR so it gets generated properly going forward: https://github.com/NixOS/nixfmt/pull/213

**Review by 0x4A6F — APPROVED** [review]

**Comment by kirillrdy** [comment]
should commit message be `nixfmt-rfc-style: unstable-2024-05-28 -> unstable-2024-07-03` otherwise ofborg will build old nifxmt ?

**Comment by kirillrdy** [comment]
ah didn't make it in time :-)

**Comment by infinisil** [comment]
Thanks, will do that next time!

**Pull request merged**